docs/use-cases/ai_workflow_automation/real-time-trading-forecasts.mdx

```diff
@@ -81,8 +81,8 @@ WHERE symbol = 'BTCUSDT'
 Next, we query for forecasts by joining the model with the Binance input data table.
 
 ```sql
-SELECT to_timestamp(cast(m.open_time as bigint)) AS open_time,
-m.open_price, m.open_price_explain
+SELECT m.open_time ,
+       m.open_price
```

Comment:
In this example, we use Lightwood, so let's undo these changes in this file.

Or, we can add `USING engine='<some nixtla model>'` in line 43.